lib/commands/interactive.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Start interactive REPL
+ */
+
+const path = require('path');
+const repl = require('../testLauncher/repl');
+const logger = require('../utils/logger');
+const {getShortFunctionBody} = require('../utils/stringUtils');
+const texts = require('../texts');
+const {stackTraceParser} = require('../utils/stackTraceParser');
+const {authContext} = require('../context');
+
+let isReplActive = false;
+
+/**
+ * Prepares parameters for the REPL session
+ * @param {{ignored: *, watch: *, repeater: Function? cwd: String?, vars: Array?}} opts
+ * @returns {*[]}
+ */
+function getReplParameters(opts) {
+	// get dir where .interactive() was called
+	const callFile = path.normalize(stackTraceParser(new Error())[2].file);
+	const cwd = path.dirname(callFile);
+	const files = opts.watch || path.join(cwd, '**/*.js');
+	const watch = Array.isArray(files)? files : [files? files : cwd];
+	const suitest = this;
```

Comment:
As I noticed `getReplParameters` is used only in `interactive` function. So `this` will be refers to `global` object? why just not do this explicitly?

Reply:
I just realized, `this` wont work with object destructuring (`const {interactive} = sutiest;`). But importing suitest here may cause circular dependency errors. Maybe it can be safely imported into repl.js.